src/patch_parser.py

```diff
@@ -352,6 +364,12 @@ def map(self, raw_line: int) -> Tuple[str, int]:
             raise IndexError(
                 'Expected ' + str(self.in_range[0]) + ' <= ' + str(raw_line) + ' <= ' + str(self.in_range[1]))
 
+    def get_range(self) -> Tuple[int, int]:
+        return self.in_range
+
+    def get_side(self) -> str:
+        return self.side
```

Comment:
these new functions seem to be unused now.
I assume this is from a previous attempt at inspecting the chunks map to find `deleted_lines` instead of storing it in InputSource.

So I think we can drop these for now.

Reply:
That's right, they are un-used. Let me drop them as well as the other un-used items.